constants/styles/MainScreen.ts

```diff
@@ -17,9 +18,10 @@ export const styles = (isDark: boolean) => StyleSheet.create({
     photoTouchableOpacity: {
         borderColor: Layout.borderColor,
         borderWidth: Layout.borderWidth,
-        height: "85%",
+        height: Dimensions.get('window').height>=650?"85%":"40%",
```

Comment:
Put this _reused_ logic into the `Layout.ts` similar to this:

```
import { Dimensions } from 'react-native';

const width = Dimensions.get('window').width;
const height = Dimensions.get('window').height;

export default {
  window: {
    width,
    height,
  },
  isSmallDevice: width < 375,
}
```

Reply:
I did not get this. Could you explain?

Reply:
Sure! I saw you are using this condition many places, I would unify it so that logic is in one place.